testnet/eth2network.Dockerfile

```diff
@@ -9,7 +9,7 @@
 # final = copies over the executables from the 'build-*' stages and prepares the final image.
 
 # Build stage for downloading dependencies based on the core defined system
-FROM golang:1.23.7-bookworm as get-dependencies
+FROM golang:1.23.5-bookworm as get-dependencies
```

Comment:
downgrade?

Reply:
I tried to move everything to 1.23.8 ( as it fixes some security issues), but ego had a different opinion. It wants 1.23.5.
I don't think this particular image matters as it's for the local eth network.